src/components/Map.js

```diff
@@ -1,56 +1,63 @@
-import React, { useContext, useState } from 'react';
-
-import GoogleMapReact from 'google-map-react';
-
-import { SearchResultsContext } from '../context/SearchResults';
-
 import Pin from './Pin';
+import GoogleMapReact from 'google-map-react';
+import { useContext, useState } from 'react';
+import { MapCenterContext } from '../context/MapCenterContext';
 
-const Map = () => {
+export const Map = ({
+  defaultCenterMap,
+  centerMap,
+  locations = [],
+  zoom = 16,
+}) => {
   const [loaded, setLoaded] = useState(false);
-  const { data } = useContext(SearchResultsContext);
-  const { centerCoords, zoom } = data;
+  const valueCenterMap = useContext(MapCenterContext);
```

Comment:
In the previous version, centerCoords and zoom were pulled out of the data object. Now, you're passing them as props and I'm not sure I understand why 😄. Also, why do you default locations to an empty array?

Reply:
@katia-utochkina These are our reasons:

1. Previous version: in this case, I think there is a little confusion before we had the `Map.js` file and it was our Map page (or view for `/` route). Now, it's a new component.
2. Empty array: maybe I'm wrong (it's possible) but some time ago I learned about 'optional' parameters, where you put some 'default' values when you don't receive some parameters with values. In this case, if we forgot to pass an array, it still works!
3. Rename files and variables: because we changed our context's name, and their values.